server/events/models/models.go

```diff
@@ -582,12 +588,14 @@ func (c CommandName) String() string {
 	switch c {
 	case ApplyCommand:
 		return "apply"
-	case PlanCommand:
+	case PlanCommand, AutoplanCommand:
 		return "plan"
 	case UnlockCommand:
 		return "unlock"
 	case PolicyCheckCommand:
 		return "policy_check"
+	case ApprovePolicyCommand:
```

Comment:
Might make more sense to make this plural as we will be support approval multiple policies by default and then eventually allowing the scope to be modified to be individual ones


`atlantis approve_policies -d <project> -p <specific_policy>` is more inuitive than
`atlantis approve_policy` (which approves all the policies across all projects)